bin/plugin/commands/build-plugins.js

```diff
@@ -0,0 +1,237 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs-extra' );
+const path = require( 'path' );
+const glob = require( 'fast-glob' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+
+exports.options = [];
+
+/**
+ * Command for copying the files required for the standalone plugin.
+ */
+exports.handler = async () => {
+	const pluginsFile = path.join( '.', 'plugins.json' );
+	fs.readFile( pluginsFile, 'utf8', ( err, jsonString ) => {
+		if ( err ) {
+			log(
+				formats.error(
+					`\nError reading file from disk: "${ err }"\n\n`
+				)
+			);
+		}
+
+		try {
+			const modules = JSON.parse( jsonString );
+			for ( const key in modules ) {
+				const pluginKey = key;
+				const pluginVersion = modules[ pluginKey ].version;
+				const pluginSlug = modules[ pluginKey ].slug;
```

Comment:
```suggestion
				const pluginVersion = modules[ pluginKey ]?.version;
				const pluginSlug = modules[ pluginKey ]?.slug;
```

@mukeshpanchal27 thank you for adding the conditional checks on these `const`'s but if they keys are missing or incorrectly spelled as per prior suggestion, even the property access would fail, hence I am suggesting the `?.` optional chaining method as well.

Reply:
Thanks @10upsimon, Updated in https://github.com/WordPress/performance/pull/662/commits/a3b2c74366d8920e39a393716313898ff6d9399c